config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
```

Comment:
I'm not sure we need ActionCable either.

Reply:
We don't 🙂

Reply:
Removed.